pkg/lib/infra/db/appdb/deps.go

```diff
@@ -73,14 +73,17 @@ func NewHandle(
 	credentials *config.DatabaseCredentials,
 	lf *log.Factory,
 ) *Handle {
+	info := db.ConnectionInfo{
+		Purpose:     db.ConnectionPurposeMain,
```

Comment:
Should global db and app db has different purpose?
The global db is used for configsource in db and always hold one connection for listening to pg pubsub.

Reply:
You're correct. I amended the commit to add `global`, and split `audit` into `audit_read_only` and `audit_read_write`